### PR TITLE
Update doc setting up development environment

### PR DESCRIPTION
**Pull request by gabechu**
### Description
Update doc for setting up development environment for redaction app.

#### Checklist
- [x] Updated **documentation** for changed code.

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+## Table of contents
+
+1. [Installing the Zendesk App Tools (ZAT)](#dev-zat)
+2. [Setting up the Python environment](#dev-python)
+3. [Lauching the Flask app](#dev-flask)
+4. [Getting browser connected to the Zendesk instance](#dev-instance)
+
+
+## Installing the Zendesk App Tools (ZAT) <a name='dev-zat'></a>
+ZAT is used for developling Zendesk apps and it bridges communications between your local (or remote) app and your Zendesk product instance.
```
Comment (arwengriffioen) [inline docs/development.md:13]: typo developing

Comment (gabechu) [reply docs/development.md:13]: Good catch, thanks! Fixed.

File: docs/development.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+# Setting Up a Development Environment
+
+
+## Table of contents
+
+1. [Installing the Zendesk App Tools (ZAT)](#dev-zat)
```
Comment (torstenrudolf) [inline docs/development.md:6]: In markdown you get a numbered list by prefixing all lines with `1. `. so if you need to insert another item you don't need to re-number. 

eg:

1. a
1. b
1. c

File: docs/development.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+2. [Setting up the Python environment](#dev-python)
+3. [Lauching the Flask app](#dev-flask)
+4. [Getting browser connected to the Zendesk instance](#dev-instance)
+
+
+## Installing the Zendesk App Tools (ZAT) <a name='dev-zat'></a>
+ZAT is used for developing Zendesk apps and it bridges communications between your local (or remote) app and your Zendesk product instance.
```
Comment (torstenrudolf) [inline docs/development.md:13]: would be good to have a reference to the ZAT doc or repository.

File: docs/development.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+```
+pip install --user poetry
+```
+Install ML libraries dependencies and Flask using `poetry`. Make sure the installation happens in the root directory.
+```sh
+poetry install
+```
+Activate the virtual environment.
+```
+poetry shell
```
Comment (torstenrudolf) [inline docs/development.md:57]: are we using a new dependency management tool here and not pipenv anymore? is it better?

File: docs/development.md (hunk 2 above)
Comment (gabechu) [reply docs/development.md:6]: Cool! I learned something new today!

File: docs/development.md (hunk 3 above)
Comment (gabechu) [reply docs/development.md:13]: Good one, link added.

File: docs/development.md (hunk 4 above)
Comment (gabechu) [reply docs/development.md:57]: It's just for this repo. I couldn't install `tensorflow` on my machine with `pipenv` when `pytorch` was there, it took forever to lock. I got switched to `poetry` and so far the locking and dependency resolution, I feel, are much better than `pipenv`.

Just took a look at `pipenv` repo, I am glad they start putting new releases. For the past one and half a years no updates made this repo which made me thought it's a dead project.

**Pull request merged**